instrumentation/apolloconfig-apolloclient/apolloconfig-apolloclient-1.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apolloconfig/apolloclient/v1_1/ApolloConfigInstrumentationModule.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apolloconfig.apolloclient.v1_1;
```

Comment:
Morning, 1.0 is supported. Since 1.1 added an interface method, Test get errors when using the 1.0 client.

I want to confirm that my module named `apolloconfig-apolloclient-1.0`
and then use
library("com.ctrip.framework.apollo:apollo-client:1.1.0")
latestDepTestLibrary("com.ctrip.framework.apollo:apollo-client:1.1.+")
Is it OK

Reply:
Sure